### DISCUSSION TITLE
'Solving PDE with any intial condition using DeepXDE"

### DISCUSSION THREAD

**Discussion by ghost** [lululxvi/deepxde#1010] [General]
Hi Lu lu,

Please I need some clarification. I am solving some parabolic equations using DeepXDE. In this problem, I am thinking of developing and training my model without any specific initial condition. In other words, I want the model to be trained so that it works for any initial condition defined on the domain. Please is this possible? If yes, do you have any idea? Thanks.

**Comment by lululxvi** [comment]
Use DeepONet.
